astroplan/constraints.py

```diff
@@ -506,15 +506,35 @@ def is_always_observable(constraints, observer, targets, times=None,
         List of booleans of same length as ``targets`` for whether or not each
         target is observable in the time range given the constraints.
     """
+    targets_is_scalar = False
+    times_is_scalar = False
+
+    if not hasattr(targets, '__len__'):
+        targets = [targets]
+        targets_is_scalar = True
+
     if not hasattr(constraints, '__len__'):
         constraints = [constraints]
 
+    if not hasattr(times, '__len__') or not hasattr(time_range, '__len__'):
```

Comment:
The first of these won't work because a scalar `Time` object still has a `__len__` - it's just that if you try to use it you get a `TypeError`.  Instead, the best thing is to do to use ``times.isscalar`` to decide if it's a scalar.  (Or maybe ``(times is not None and times.isscalar)``?  I think ``times`` can be None if you give a ``time_range``, right?)